### DISCUSSION TITLE
World file path raspberry pi

### DISCUSSION THREAD

**Discussion by JOLIGEOLIFANJTES** [Pryaxis/TShock#2961] [General]
i am not able to find the path where tshock on the py stores the worlds.
Could i get some help with this?|

Thanks!

**Comment by 0x3fcf1bbd** [comment]
It should be stored in `~/.local/share/Terraria/Worlds/`.

**Reply by JOLIGEOLIFANJTES** [reply]
Thanks! helped a lot

**Comment by hakusaro** [comment]
You can use the /worldinfo command to see where the current world is stored.

**Reply by JOLIGEOLIFANJTES** [reply]
owh nice thats also good to know i am just starting with these kind of things
